external/craigslist/fetch.py

```diff
@@ -0,0 +1,7 @@
+from craigslist import CraigslistHousing
+from typing import Dict, List
+
+
+def fetch_craigslist_housing(*, limit, **kwargs) -> List[Dict[str, any]]:
```

Comment:
For the first `*` parameter in this, doesn't it still need to have a parameter name in order to be able to call it later? i.e. `def fetch_craigslist_housing(*args, ...`?

Reply:
I wanted to set `limit` as a required parameter and also with the keyword `limit`. In other words, I wanted to prevent `fetch_craigslist_housing(3)` instead of using `fetch_craigslist_housing(limit=3)`. 
And for the other keyword params, I wanted to make them identical with the params we pass to the Craigslist framework.

There was no strong reason  for doing this but with this function, we will use a lot of keyword parameters and I just wanted to give `limit` a similar tone and manner.

For the keyword only arguments, I referred to this doc:
https://www.python.org/dev/peps/pep-3102/#id4

Reply:
Ahhh I see, I mistook the `*` for something else.  

Given that info, I have a different question. If I wanted to pass in a zipcode to limit the search, how would I do that? Or is this particular function call not meant to support that?